ledger-service/http-json/src/itlib/scala/http/AbstractWebsocketServiceIntegrationTest.scala

```diff
@@ -747,6 +747,28 @@ abstract class AbstractWebsocketServiceIntegrationTest
     go(createCount)
   }
 
+  private[this] def readAcsAt(until: domain.Offset): Consume.FCC[JsValue, Map[String, JsValue]] = {
+    val dslSyntax = Consume.syntax[JsValue]
+    import dslSyntax._
+    def go(
+        acs: Map[String, JsValue],
+        latest: Option[domain.Offset],
+    ): Consume.FCC[JsValue, Map[String, JsValue]] =
+      if (latest.fold(false)(domain.Offset.ordering.greaterThanOrEqual(_, until))) {
+        point(acs)
+      } else {
+        for {
+          ContractDelta(creates, deletes, offset) <- readOne
+          _ = println("#" * 80)
+          _ = println(offset)
+          _ = println(creates)
+          _ = println(deletes)
```

Comment:
Debug prints, will be removed before moving this out of draft.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/9847/commits/6a832849983f8d68bdbc48b8ede4b12bd84f16be